### DISCUSSION TITLE
Feature suggestion : option to start YAKD with its window of settings minimized.

### DISCUSSION THREAD

**Discussion by PiorGit** [Jagailo/YetAnotherKeyDisplayer#35] [Ideas]
Hello,
I've just started using YAKD and it is looking excellent.

One thing I find somewhat annoying though is the way the software starts up with its window of options open. This behavior was actually quite useful at first as it made the initial embarking very easy ; but once things are setup it does become  a hindrance.

Therefore, it would be great to have a tickbox enabling "Program starts with its window of options minimized". That way a user familiar enough with the software (after a few initial uses) would only bringing up the panel when really needed, as opposed to seing it and having to minimize it every time. As a matter of fact most screen display tools do that by default, only having their panel of options accessible/expandable/maximizable through the System Tray. But there's no right or wrong way IMHO.

Of course this doesn't ends there as there is also the subtlety of having the option to minimize to the Tray as opposed to remaining in the Taskbar. But that's another (more minor) topic altogether, and it is covered in other discussions.
 
 Thank you for this useful tool.

**Comment by Jagailo** [comment]
At least in version 1.5 the program is completely hidden in the tray
